### PR TITLE
LSR-1068 Adapt to the new version of Mobile SDK

### PR DESCRIPTION
**Pull request by dgarciasarai**
~This PR depends on `liferay-mobile-sdk` PR [#41](https://github.com/liferay/liferay-mobile-sdk/pull/41)~ (Fixed)

~Now depends on https://github.com/liferay/liferay-screens/pull/522~ (Fixed)

### CODE REVIEW

File: ios/Framework/Core/Context/SessionContext.swift [hunk 1]
```diff
@@ -302,9 +288,6 @@ open class SessionContext: NSObject {
 		guard let result = storage.load(shouldLoadServer: shouldLoadServer) else {
 			return false
 		}
-		guard result.session.server != nil else {
-			return false
-		}
```
Comment (victorg1991) [inline ios/Framework/Core/Context/SessionContext.swift]: why?

Comment (victorg1991) [reply ios/Framework/Core/Context/SessionContext.swift]: Aaah, It can't be nil now, right?

Comment (dgarciasarai) [reply ios/Framework/Core/Context/SessionContext.swift]: That's it

**Comment by victorg1991** [comment]
Good job, thanks! :D

**Pull request merged**